README.md

```diff
@@ -23,8 +23,8 @@ The autograding effort is broken into three main parts:
 ## Installation
 
 This project uses Go 1.21.
-Development and deployment are intended for a Linux environment,
-but all Go code is written in a platform independent fashion.
+Development and deployment are intended for a Linux environment.
+Temporary file management relies on POSIX (i.e Linux, MacOS, WSL) conventions 
```

Comment:
The extra details here (temp file management) are incorrect.
This is a case where you can just be more general, and create something that is both more accurate and generally better.